src/backend/base/langflow/components/helpers/ParseJSONData.py

```diff
@@ -0,0 +1,60 @@
+from langflow.custom import Component
+from langflow.inputs import HandleInput, MessageTextInput
+from langflow.io import Output
+from langflow.schema import Data
+from langflow.schema.message import Message
+import jq
```

Comment:
Does jq work on Windows?

Reply:
yeah pip should take care of it
https://pypi.org/project/jq/